cpp/include/cudf/fixed_point/fixed_point.hpp

```diff
@@ -661,7 +645,7 @@ CUDA_HOST_DEVICE_CALLABLE fixed_point<Rep1, Rad1> operator-(fixed_point<Rep1, Ra
 
   release_assert(
     !subtraction_overflow<Rep1>(lhs.rescaled(scale)._value, rhs.rescaled(scale)._value) &&
-    "fixed_point overflow of underlying representation type " + print_rep<Rep1>());
+    "fixed_point overflow");
```

Comment:
wrapping these in `__CUDACC_DEBUG__` kinda defeats the point of _release_ assert. Might as well just use normal `assert`.

Reply:
Done.